docs/documentation/release_notes/topics/26_2_0.adoc

```diff
@@ -63,3 +63,11 @@ link:{grafanadashboards_link}[The guide] contains two dashboards.
 Because the https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-XSS-Protection[`X-XSS-Protection` header] is no longer supported by any user agents that are supported by Keycloak, it has been removed. This header was a feature of Internet Explorer, Chrome, and Safari that stopped pages from loading when they detected reflected cross-site scripting (XSS) attacks.
 
 We don't expect that this will impact any deployments due to the lack of support in user agents, as well as this feature being supplanted by https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP[Content Security Policy (CSP)].
+
+= Enable token based authentication for SMTP (XOAUTH2)
+
+The Keycloak outgoing SMTP mail configuration now supports token authentication (XOAUTH2).
+
+Many service providers (Microsoft, Google) are moving towards SMTP OAuth authentication and end the support for basic authentication.
+
+The token is gathered using Client Credentials Grant.
```

Comment:
I think we can remove the white space here. Let's also add a link at the end to the documentation you write explaining this feature in further detail and keep implementation details there.

```suggestion
The Keycloak outgoing SMTP mail configuration now supports token authentication (XOAUTH2). Many service providers (Microsoft, Google) are moving towards SMTP OAuth authentication and are ending support for basic authentication.
```